hw2/04-background.js

```diff
@@ -1 +1,27 @@
 // Add your code here
+
+const colors = ['#FF0000', '#FFA500', '#FFFF00', '#00FF00', '#0000FF', '#A020F0', '#FFC0CB']
+
+async function startColorChange() 
+{
+   let interval = document.getElementById("interval");
+
+   document.getElementById("start").hidden = true;
+   document.getElementById("stop").hidden = false;
+
+   while(document.getElementById("stop").hidden == false)
```

Comment:
For comparison operators use strict equality as recommended by the [AirBnB style guide](https://github.com/airbnb/javascript#comparison--eqeqeq).